testsuite/tests/parallel/domain_parallel_spawn_burn_gc_set.ml

```diff
@@ -0,0 +1,63 @@
+(* TEST
+* hasunix
+include unix
+** bytecode
+** native
+*)
+
+open Domain
+
+(* This test looks to spawn domains while doing a bunch of explicit minor and
+   major GC and Gc.set calls from parallel domains. *)
+
+let rec set_gc l =
+  if l > 16 then ()
+  else
+    let g1 = Gc.get() in
+      Gc.set { g1 with
+        minor_heap_size = ((l mod 4) + 1) * (1 lsl 18);
+      };
+      set_gc (l + 1)
+
+let test_size =
+  try int_of_string (Sys.getenv "OCAML_TEST_SIZE")
+  with Not_found | Failure _ -> 0
+
+let (list_size, num_domains, niters) =
+  if test_size >= 2 then (14, 25, 20) else (13, 12, 5)
```

Comment:
I appreciate the parameterization by `OCAML_TEST_SIZE`.  Still, this test starts much more domains than available processor cores (2, 4, or 8 depending on `OCAML_TEST_SIZE`).  Is this wise?  Is this needed?

Reply:
Very good point, indeed a smaller test size will do! Fixed.